src/main/java/com/google/growpod/data/Garden.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.growpod.data;
+
+/** Garden data class. */
+public class Garden {
+
+  /** A unique id. */
+  private String id;
+
+  private String name;
+
+  /** Latitude of garden. */
+  private double lat;
+
+  /** Longitude of garden. */
+  private double lng;
+
+  /**
+   * Foreign Key to this garden's administrator. Currently, there is no checking as to whether this
```

Comment:
nit: The annotation `Currently, there is no checking as to whether this is a valid key.` could be made more precise here. Specifically, it is hinting at two things: (1) There are valid and invalid keys, and (2) this isn't checked but the presence of "currently" suggests this is intended to change. 

One option is to simply annotate as is: `This key could be valid or invalid,` and possibly give examples -- do you mean invalid in that it could be the wrong type or even `null`? Or do you mean invalid in that it simply doesn't correspond to anything in the database?  In either case, how might an invalid key get generated?

Another option is to leave a more explicit note: `// TODO(username or issue#): Check validity.` You can leave more info in the comment or the corresponding issue to help leave a paper trail for why this is needed and what it entails.

Reply:
Will nix `currently`.

`adminId`, when the program is working properly, will always contain a valid user id and will not be `null`. What I meant to say was that, if you passed in an invalid user id, that this class would not perform any checking on this value.

However, I will replace this annotation with a clearer one: `Must be a valid user key`.